### ISSUE TITLE
Toolwindow icon is not visible in IDE light mode

### ISSUE THREAD

**Issue by kedar-joshi** [JetBrains/HotSpotCrashExaminerPlugin#16]
In light mode toolwindow icon is pretty much invisible.

![image](https://github.com/user-attachments/assets/c7ce34e5-b796-4237-af5b-585718c9f343)

**Environment**
```
IntelliJ IDEA 2024.3 RC (Ultimate Edition)
Build #IU-243.21565.129, built on November 7, 2024
Subscription is active until January 19, 2025.
Runtime version: 21.0.5+8-b631.16 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.awt.windows.WToolkit
Windows 10.0
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 8192M
Cores: 8
Registry:
  idea.coverage.new.test.tracking.enabled=false
  new.project.load.remote.templates=true
  ide.allow.merge.buttons=false
  ide.experimental.ui=true
  ide.slow.operations.assertion=false
  ide.slow.operations.assertion.action.update=false
  ide.slow.operations.assertion.generic=false
  ide.slow.operations.assertion.other=false
  ide.ui.new.file.chooser=true
  coroutine.scope.model=false
  i18n.locale=
  terminal.new.ui=true
Kotlin: 243.21565.129-IJ
````

**Comment by mkartashev** [comment]
Thanks for reporting this! I will add a proper light-mode icon.

**Comment by mkartashev** [comment]
Should be fixed in 0.0.3
